camel/benchmarks/apibank.py

```diff
@@ -465,7 +465,7 @@ def __init__(self, samples: List[APIBankSample]):
         # Place holder for import as the import
         # only works after the files have been downloaded
         try:
-            from api_bank.tool_manager import (  # type: ignore[import-not-found]
+            from api_bank.tool_manager import (  # type: ignore[import-untyped]
```

Comment:
If the file is not downloaded, there will be a `import-not-found` error which will not be covered by `ignore[import-untyped]`